app/src/androidTest/java/fr/free/nrw/commons/NearbyActivityTest.java

```diff
@@ -24,7 +31,34 @@
 
     @Test
     public void testActivityLaunch() {
+        onView(withText(R.string.gps_disabled))
+                .check(matches(isDisplayed()));
+        onView(withId(android.R.id.button2)).perform(click());
         onView(withText(R.string.title_activity_nearby))
-                .check(ViewAssertions.matches(isDisplayed()));
+                .check(matches(isDisplayed()));
     }
+
+//    private void turnGPSOn(Context context){
```

Comment:
Could I ask why this is commented out?

Reply:
I was trying out something. It doesn't matter now. We have @psh's test to replace ours.